themes/ddbasic/sass/components/view/view.scss

```diff
@@ -66,11 +66,13 @@
     // Styleguide 2.1.9
 
     // Small grid
+    @at-root .grid-small#{&},
```

Comment:
It is hard to tell what is going on here without any comments but as far as I can tell you are trying to make the `.grid` classes usable without the dependency on Views markup by using [`@at-root`](http://sassbreak.com/getting-back-to-our-roots/).

I think this is the wrong way to go about something as fundamental as a grid system. It convolutes the structure of the SCSS.

If the grid system should be usable in general then it should be refactored as a general implementation which *then* can be used with Views using mixins, extends or some other method.